app/views/messages/_header.html.erb

```diff
@@ -1,10 +1,9 @@
 <h1 class="mt-3 text-center text-white"><%= @student.last_name %>, <%= @student.first_name %>'s<br>Correspondence Notebook</h1>
   <p class="text-center text-white mb-3">Student Number: <%= @student.student_number %></p>
-  <div class="text-center">
-    <%= link_to "Add New Message", new_student_message_path, class: "btn btn-primary" if current_user.teacher%>
```

Comment:
Don't we want a button that will reveal the form instead of having it always there?

Reply:
Yes, that is a good idea! This can happen in a separate branch though. But let's discuss this with the others to see what they think